BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -148,6 +155,8 @@ public class Plugin : IPlugin {
 				AddSubscriber(gamePauseManager, "_gameDidPauseSignal", OnGamePause);
 				// private GameEvent GamePauseManager#_gameDidResumeSignal
 				AddSubscriber(gamePauseManager, "_gameDidResumeSignal", OnGameResume);
+				// public ScoreController#noteWasMissedEvent<float energy>
```

Comment:
The correct comment here would be
```suggestion
				// public GameEnergyCounter#gameEnergyDidChangeEvent<float energy>
```